packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1207,7 +1134,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     }
 
     protected async getDataStore(id: string, wait = true): Promise<IFluidRouter> {
```

Comment:
Should this move too...?

Reply:
i think there are back-compat concerns here, as i think @vladsud has bohemia casting to any to get at this

Reply:
hence the protected, rather than the prive

Reply:
Ok yeah I was wondering about the protected.  Makes sense.